internal/tofu/node_resource_apply.go

```diff
@@ -30,6 +31,21 @@ var (
 func (n *nodeExpandApplyableResource) expandsInstances() {
 }
 
+// CreateBeforeDestroy implementation for GraphNodeDestroyerCBD
+func (n *nodeExpandApplyableResource) CreateBeforeDestroy() bool {
```

Comment:
That seems like a miss during the initial implementation, since we follow the same approach for `nodeExpandPlannableResource`. Good catch!